File: slicing-ios/SlicingGame/Infrastructure/Inflator/Inflators.swift
//
//  Inflators.swift
//  Inflator: a list of json inflators, like viewlet creator
//

import JsonInflator

class Inflators {
    
    static let module = JsonInflator(inflatableKey: "module")
    static let viewlet = JsonInflator(inflatableKey: "viewlet", attributeSetKey: "viewletStyle")

}

File: slicing-android/app/src/main/java/com/crescentflare/slicinggame/infrastructure/inflator/Inflators.kt
package com.crescentflare.slicinggame.infrastructure.inflator

import com.crescentflare.jsoninflator.JsonInflator

/**
 * Inflator: a list of json inflators, like viewlet creator
 */
object Inflators {

    val module = JsonInflator("module")
    val viewlet = JsonInflator("viewlet", "viewletStyle")

}
